teamA/gisuk/app.js

```diff
@@ -31,6 +31,134 @@ app.use(express.json());
     
 app.get("/ping", (req,res) => {
   res.status(200).json({"message" : "pong"});
+});
+
+app.get("/user", async (req,res) => {
+  await appDataSource.query(
+    `SELECT 
+      users.name,
+      users.email,
+      users.profile_image
+    FROM users`,
+    (err, rows) => {
+      res.status(200).json(rows);
+    })
+});
+
+app.get("/user/post/:user_id", async (req,res) => {
+  const user_id = req.params.user_id;
+  await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.profile_image as userProfileImage,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent 
+    FROM posts INNER JOIN users on posts.user_id = users.id 
+    WHERE users.id LIKE ${user_id};`,
+    (err, rows) => {
+      const posting = [];
+      rows.map((el) => {
+        posting.push({
+          postingId : user_id, 
+          postingTitle : el.postingTitle, 
+          postingContent : el.postingContent
+        })
+      })
+    result = {
+      userId : user_id, 
+      userProfileImage : rows[0].userProfileImage, 
+      postings : posting
+    }
+    res.status(200).json({data : result});
+    }
+  )
+})
+
+app.get("/post", async (req,res) => {
+    await appDataSource.query(
+    `SELECT
+      users.id as userId,
+      users.profile_image as userProfileImage,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+    FROM posts INNER JOIN users on posts.user_id = users.id;`,
+    (err, rows) => {
+      res.status(200).json({data :rows});
+  })
+})
+
+app.post("/user", async (req, res, next) => {
+  const { name, email, profile_image, password} = req.body;
```

Comment:
conding의 naming style에는 CamelCase 와 snake_case가 있습니다.

node.js는 camelCase를 사용하며 javacript의 또한 필수적인것은 아니지만 대체로 CamelCase를 사용하고 있습니다.

다른 코드들과 통일성을 위해 요청의 바디에 들어오는 정보들을 구조 분해 할당을 통해 변수로 넣어줄 때도 CamelCase를 사용하여 주세요!
```suggestion
  const { name, email, profileImage, password} = req.body;
```